nmdc_runtime/site/ops.py

```diff
@@ -680,6 +683,13 @@ def translate_portal_submission_to_nmdc_schema_database(
     metadata_submission: Dict[str, Any],
     omics_processing_mapping: List,
     data_object_mapping: List,
+    study_category: Optional[str],
+    study_doi_category: Optional[str],
+    study_doi_provider: Optional[str],
+    study_funding_sources: Optional[List[str]],
+    study_pi_image_url: Optional[str],
+    biosample_extras: Optional[list[dict]],
+    biosample_extras_slot_mapping: Optional[list[dict]],
```

Comment:
Note to self (@eecavanna):
- Consider that list typehints are `list` in some places and `List` (from `typing`) in other places
- `list` became usable as of some recent version of Python (making `typing.List` unnecessary)
- Look at what Python version this project's docs indicates it requires